isb_lib/sitemaps/sitemap_fetcher.py

```diff
@@ -8,40 +9,89 @@
 import typing
 import logging
 
+import re
 import isb_lib.core
+import json
 from isb_lib.models.thing import Thing
 
 
 class ThingFetcher:
     pass
 
 
+class ThingsFetcher:
+    pass
+
+
+IDENTIFIER_REGEX = re.compile(r".*/thing/(.*)")
+
+
```

Comment:
This is the new batch fetch mechanism.  I will double check, but I think I can remove the previous ThingFetcher implementation.